invesalius/gui/task_navigator.py

```diff
@@ -1991,8 +1989,10 @@ def OnLinkBrain(self, event=None):
             slic = sl.Slice()
             prj_data = prj.Project()
             matrix_shape = tuple(prj_data.matrix_shape)
+            spacing = tuple(prj_data.spacing)
+            img_shift = spacing[1] * (matrix_shape[1] - 1)
             self.affine = slic.affine.copy()
```

Comment:
If slice.affine is None this error happens:

```
Traceback (most recent call last):
  File "/home/thiago/Sources/github/invesalius3/invesalius/gui/task_navigator.py", line 1994, in OnLinkBrain
    self.affine = slic.affine.copy()
AttributeError: 'NoneType' object has no attribute 'copy
```

Reply:
Affine now is identity by default. Should work, can tou try again please?

Reply:
It's working now. thanks!